webapp-django/crashstats/base/jinja2/crashstats_base.html

```diff
@@ -155,6 +155,7 @@ <h1>Product Navigation</h1>
             <ul>
                 <li>
                 {% if request.user and request.user.is_active %}
+                    <span>{{ request.user.email }}</span>
                     <a href="{{ url('profile:profile') }}"
```

Comment:
A pipe here between the email and the other links would help visually separate things a tiny bit and would be consistent with the rest of the footer links.